### DISCUSSION TITLE
Adding DUO integration

### DISCUSSION THREAD

**Discussion by jaypfe** [Security-Onion-Solutions/securityonion#11512] [Unsupported Versions] [answered]
Can I make a formal request to add the DUO integration to the officially supported stack?

**Comment by dougburks** [comment, marked as answer]
The cisco_duo integration should be included in the upcoming 2.4.30 release:
https://github.com/Security-Onion-Solutions/securityonion/issues/11513
https://docs.securityonion.net/en/2.4/release-notes.html
https://docs.securityonion.net/en/2.4/elastic-fleet.html#integrations
